packages/web-client/app/utils/validation.ts

```diff
@@ -0,0 +1,66 @@
+import BN from 'bn.js';
+import { toBN, toWei } from 'web3-utils';
+
+// token input validations that assume string inputs are in wei
```

Comment:
I just realised this comment is incorrect - it should assume string inputs are in ether.
```suggestion
// token input validations that assume string inputs are in ether
```